alembic/versions/f63e43dcf0ca_add_tenant_to_subscription.py

```diff
@@ -0,0 +1,29 @@
+"""add tenant to subscription
+
+Revision ID: f63e43dcf0ca
+Revises: 35b30e165f6
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'f63e43dcf0ca'
+down_revision = '35b30e165f6'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column(
+        'webhookd_subscription',
+        sa.Column(
+            'tenant_uuid',
+            sa.String(36),
+            nullable=False,
+            server_default=sa.text('00000000-0000-0000-0000-000000000000'),
+        )
+    )
+    op.alter_column('webhookd_subscription', 'tenant_uuid', server_default=None)
+
+def downgrade():
+    pass
```

Comment:
We try to make downgrade when the impact is the schema and not the data. It's generally easy to write and easy for the reviewer :smile:

Reply:
done